econml/dml/_rlearner.py

```diff
@@ -401,11 +403,23 @@ def rlearner_model_final_(self):
 
     @property
     def models_y(self):
-        return [mdl._model_y for mdl in super().models_nuisance_]
+        models_y = []
+        for mdls in super().models_nuisance_:
+            models_y_eachmc = []
+            for mdl in mdls:
+                models_y_eachmc.append(mdl._model_y)
+            models_y.append(models_y_eachmc)
+        return models_y

```

Comment:
I think this can just be:
```suggestion
        return [[mdl._model_y for mdl in mdls] for mdls in super().models_nuisance_]
```

Reply:
Good point! I will change all of it.